src/Dialog/formComponents/fields/ObjectField.js

```diff
@@ -180,6 +180,15 @@ class ObjectField extends Component {
               </Button>
             </div>
           )}
+          {!title && (
```

Comment:
The only difference between buttons when `title` exists and not exists is button text, so please move condition next to plus icon (`Add${!title ? '' : ' ' + title}`) and make one button.